### DISCUSSION TITLE
debian 12 (bookworm) empty repository

### DISCUSSION THREAD

**Discussion by badsdv** [piraeusdatastore/piraeus-operator#630] [Q&A] [answered]
Hi
There are no images for debian 12 (bookworm) on https://quay.io/.
When starting a cluster on satellite nodes, an error loading the image appears.
After looking at the repository where the download was coming from, I found that it was empty.
https://quay.io/repository/piraeusdatastore/drbd9-bookworm .

**Comment by WanzenBug** [comment, marked as answer]
I triggered a rebuild, looks like we forgot to set up the permissions on the new repository. Should be available in ~1 hour.

**Reply by WanzenBug** [reply]
Images are up now.

**Reply by badsdv** [reply]
Thank you
